### PR TITLE
yahoo find league id for all years

### PR DESCRIPTION
**Pull request by brian-dev8**
fix issue #27

### CODE REVIEW

**Comment by joeyagreco** [comment]
Thanks for looking into this!

I was investigating this a bit earlier this week and opened this [issue](https://github.com/mattdodge/yahoofantasy/issues/28) in the library that I'm using for Yahoo.

The owner of that repo didn't think there was a link that Yahoo returned to find the previous/next league ID, but your find may be just that.

Have you tested this locally with a multi-year Yahoo league? 

I would test myself but I do not have access to any Yahoo leagues that have multiple years in them.

File: example/league_loader/yahooLeagueLoaderExample.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 # The app will need to have a callback/redirect URI of https://localhost:8000
 
 # Get a League object with years 2019 and 2020 for Yahoo league with ID: 123456.
+# ID needs to be from the oldest year given, and then it will figure out the ID's for the newer years
```
Comment (joeyagreco) [inline example/league_loader/yahooLeagueLoaderExample.py:10]: NIT: change "ID's" to "IDs"

File: leeger/league_loader/YahooLeagueLoader.py [hunk 2]
```diff
@@ -59,11 +59,13 @@ def loadLeague(self) -> League:
             raise TimeoutError("Login to yahoofantasy timed out.")
         ctx = Context()
         yahooLeagues = list()
+        curYearLeagueId = self._leagueId
```
Comment (joeyagreco) [inline leeger/league_loader/YahooLeagueLoader.py:62]: can we use [snake case](https://khalilstemmler.com/blogs/camel-case-snake-case-pascal-case/#Snake-case) for variables in this PR for consistency?

File: leeger/league_loader/YahooLeagueLoader.py [hunk 3]
```diff
@@ -169,3 +171,12 @@ def __getOwnerByYahooTeam(self, yahooTeam: YahooTeam) -> Owner:
             return self.__yahooManagerIdToOwnerMap[yahooManagerId]
         raise DoesNotExistException(
             f"Yahoo Manager ID {yahooManagerId} not found in saved Yahoo Manager IDs.")
+
+    # First 3 digits are the game_key, followed by the league_id which is typically 5-6 digits
```
Comment (joeyagreco) [inline leeger/league_loader/YahooLeagueLoader.py:175]: NIT: Love the comment for clarity, can we move it inside the method as a doc string?

Just in between """ """

File: leeger/league_loader/YahooLeagueLoader.py (hunk 2 above)
Comment (brian-dev8) [reply leeger/league_loader/YahooLeagueLoader.py:62]: I can, it just seemed like variables introduced locally were using camel case, and only the built in ones from the yahoo api objects were snake lol.

**Comment by brian-dev8** [comment]
> Thanks for looking into this!
> 
> I was investigating this a bit earlier this week and opened this [issue](https://github.com/mattdodge/yahoofantasy/issues/28) in the library that I'm using for Yahoo.
> 
> The owner of that repo didn't think there was a link that Yahoo returned to find the previous/next league ID, but your find may be just that.
> 
> Have you tested this locally with a multi-year Yahoo league?
> 
> I would test myself but I do not have access to any Yahoo leagues that have multiple years in them.

Yes, I have one that goes back to 2010 I tested with. 😃

File: leeger/league_loader/YahooLeagueLoader.py (hunk 2 above)
Comment (joeyagreco) [reply leeger/league_loader/YahooLeagueLoader.py:62]: I apologize, I got my projects mixed up.

This library does indeed use camel case, you did this correctly.

**Comment by joeyagreco** [comment]
As of [version 1.3.1](https://github.com/joeyagreco/leeger/releases/tag/v1.3.1), this issue has been fixed.

Thank you for your help on this.

**Pull request closed without merging**